README.md

```diff
@@ -1,31 +1,65 @@
 # Oxford WT Python Workshop
```

Comment:
So my view in the future is to move the "python" part of the readme to a sub-readme under `tutorials/python`. And then have an overarching readme for the entire course.

@philbiggin how would you feel about moving all the material to github?